Automattic/wp-calypso
# A4A: Fixing minor button styles

## Proposed Changes

* This PR adds two minor style fixes, as follow-ups to the style changes that were brought about by merging this PR: https://github.com/Automattic/wp-calypso/pull/89414#top
* Those fixes include the Boost action buttons on the Boost preview pane in the sites dashboard, and the license action buttons on the license page, within the expanded content sections.

Design specs (sites dashboard): 1UpoqS1KkCtLXVX68TbMSM-fi-6210_94604

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally.

* Visit `http://agencies.localhost:3000/purchases/licenses`. Expand the section for a product, and hover over one of the two buttons (view site or Debug site). The background color should be a light grey, not black as it is in trunk.
* Visit `http://agencies.localhost:3000/sites`, and open the preview pane for a site, where the Boost pane shows the 'Upgrade to auto-optimize' and 'Settings' button. The font color on the Settings button is only marginally different - `#021a23` vs black.


## Pre-merge Checklist



- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)? PR
